shm/inc/ship.hpp

```diff
@@ -21,8 +23,15 @@ class Ship {
     size_t getSpeed() const { return speed_; }
     std::string getName() const { return name_; }
     size_t getId() const { return id_; }
+    Cargo getCargo(size_t index) const{
+        if(index <  0){
+            std::cerr << "Invalid index\n";
+        }
+        return cargo_[index];
+    }

```

Comment:
I think we should move the declaration of that function to cpp file. Additionally, we should check if the given index is not greater or equal `cargo_` size.